sasmodels/convert.py

```diff
@@ -92,8 +92,8 @@ def _rescale_sld(model_info, pars, scale):
 
 
 def _get_translation_table(model_info, version=(3,1,2)):
-    _, translation = CONVERSION_TABLE.get(version, {}).get(model_info.id, [None, {}])
-    translation = translation.copy()
+    dict = CONVERSION_TABLE.get(version, {}).get(model_info.id, [None, {}])
+    translation = dict[1].copy()
```

Comment:
Little risky using dict as a variable name, particularly when it contains a list.